app/test_split_integer.py

```diff
@@ -2,20 +2,35 @@
 
 
 def test_sum_of_the_parts_should_be_equal_to_value() -> None:
-    pass
+    value = 32
+    number_of_parts = 6
```

Comment:
I think you should not create these variables, you can write it all in one line

Example:
```
assert sum(split_integer(32, 6)) == 32
```

Please fix it here in other places